cmd/timestamp_check/types.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"time"
+
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/binance"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/bitfinex"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/bitflyer"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/bithumb"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/bitmex"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/bitstamp"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/bittrex"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/btcmarkets"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/btse"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/coinbasepro"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/coinbene"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/coinut"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/exmo"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/gateio"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/gemini"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/hitbtc"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/huobi"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/itbit"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/kraken"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/lakebtc"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/lbank"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/localbitcoins"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/okcoin"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/okex"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/poloniex"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/yobit"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/zb"
+)
+
+const defaultSleepTime = time.Minute
+
+// SupportedExchanges exchanges are supported exchanges
+var SupportedExchanges = []exchange.IBotExchange{
```

Comment:
Relates to my previous nit, but you can use exchange.Exchanges and in conjunction with a basic engine, you can use LoadExchange (will call SetDefaults & Setup automatically, plus you can use a sync.WaitGroup) to load each of these individually so you don't have to hard code all of these in plus have support for future exchange additions